src/packages/members/member-group/components/input-member-group/input-member-group.context.ts

```diff
@@ -4,8 +4,11 @@ import { UMB_MEMBER_GROUP_PICKER_MODAL } from '../member-group-picker-modal/memb
 import { UmbPickerInputContext } from '@umbraco-cms/backoffice/picker-input';
 import type { UmbControllerHostElement } from '@umbraco-cms/backoffice/controller-api';
 
-export class UmbMemberPickerContext extends UmbPickerInputContext<UmbMemberGroupItemModel> {
+export class UmbMemberGroupPickerContext extends UmbPickerInputContext<UmbMemberGroupItemModel> {
```

Comment:
I have come to realize that we will at some point end in a name collision when we start implementing a "Picker" extension because these class names aren't as specific as the class they extend. So I think we should use this opportunity to make it a bit more explicit and call it `UmbMemberGroupPickerInputContext`.

see #https://github.com/umbraco/Umbraco.CMS.Backoffice/pull/2206 for more context

Reply:
I've updated all the class names that extend from the `UmbPickerInputContext` class.